lib/rules/at-rule-no-unknown/__tests__/index.mjs

```diff
@@ -96,6 +96,10 @@ testRule({
 		{
 			code: '@scroll-timeline foo {}',
 		},
+		{
+			code: '@color-profile --swop5c {}',
+			skip: true,
```

Comment:
[ask] Why skipped? Even if there is no browser support, is adding `@color-profile` a problem?

Reply:
The problem is that we don't have a policy regarding unimplemented features.
#7510 is not even resolved after all. This will do for now.
i.e. if it was behind a flag the addition could be considered

see also Fyrd/caniuse#5958

Reply:
Okay, so can you remove the case to avoid any confusion? If it's requested, then we can add it again.